metrics/network/remote_network/direct-remote-networking-iperf3.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script will measure network bandwidth with iperf3 tool using a
```

Comment:
It's not immediately obvious to me how this test is different from the existing https://github.com/clearcontainers/tests/blob/master/metrics/network/remote_network/remote-networking-iperf3.sh - can you elaborate?